csvy/validators/csv_dialect.py

```diff
@@ -127,7 +41,7 @@ class CSVDialectValidator(BaseModel):
 
     delimiter: str = Field(default=",")
     doublequote: bool = Field(default=True)
-    escapechar: Optional[str] = Field(default=None)
+    escapechar: str | None = Field(default=None)
```

Comment:
Might be clearer to explicitly put the default value here, e.g.:

```suggestion
    escapechar: str = Field(default="\\")
```

Reply:
Actually, the default accoriding to the specification is not set it https://specs.frictionlessdata.io/csv-dialect/#specification

Reply:
And now that I check the specification, I'm missing several fields... 😢

Reply:
Made adding the missing fields into another issue.